Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1824,6 +1850,17 @@ extension BrowserViewController: ReaderModeDelegate, UIPopoverPresentationContro
     }
 }
 
+// MARK: - UIPopoverPresentationControllerDelegate
+
+extension BrowserViewController: UIPopoverPresentationControllerDelegate {
+    func popoverPresentationControllerDidDismissPopover(popoverPresentationController: UIPopoverPresentationController) {
+        displayedPopoverController = nil
```

Comment:
`displayedPopoverController` could indeed be a weak reference and this would work. However we also need the `updateDisplayedPopoverProperties` reference and this one cannot be weak as BrowserViewController is the owner of this property and no other object as a reference to it. 

I don't think about another way of doing this. If you do, let me know :)